packages/backend/src/api/controllers/tvl/DetailedTvlController.ts

```diff
@@ -44,25 +45,25 @@ export class DetailedTvlController {
   async getDetailedTvlApiResponse(): Promise<
     DetailedTvlApiResponse | undefined
   > {
-    const minimumTimestamp = await this.getMinimumTimestamp()
+    const dataTimings = await this.getDataTimings()
 
-    if (!minimumTimestamp) {
+    if (!dataTimings.latestTimestamp) {
       return
     }
```

Comment:
Do we want to return undefined (404) is data is out sync now or later?

Reply:
yeah, let's do it, we will observe how it behaves on staging, but hide it behind the feature flag

Reply:
https://github.com/l2beat/l2beat/pull/1776/commits/96dab8887c3ff7a92dc428953f4701a77e660ce0